marm-mcp-server/marm_mcp_server/endpoints/logging.py

```
@@ -45,14 +45,10 @@ async def marm_log_session(request: SessionRequest):
         }
     except sqlite3.Error as e:
         print(f"Database error in marm_log_session: {e}")
-        raise HTTPException(
-            status_code=500, detail="Database error during session creation."
-        )
+        return {"status": "error", "message": f"Database error during session creation: {str(e)}"}
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)